### ISSUE TITLE
Sampler App crashes on first launch 

### ISSUE THREAD

**Issue by iadnan** [ConnectSDK/Connect-SDK-Cordova-API-Sampler#2]
The sampler app crashes when launched for the first time

**Comment by eunikolsky** [comment]
The exception is:

```
2015-08-20 10:52:50.825 Connect-SDK-Cordova-API-Sampler[1643:381442] starting discovery from cordova
2015-08-20 10:52:50.825 Connect-SDK-Cordova-API-Sampler[1643:381442] *** Assertion failure in -[DiscoveryManager setPairingLevel:], /Users/eugene/proj/Connect-SDK-iOS/core/Discovery/DiscoveryManager.m:446
2015-08-20 10:52:50.826 Connect-SDK-Cordova-API-Sampler[1643:381442] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Cannot change pairing level while DiscoveryManager is running.'
*** First throw call stack:
(0x1822842d8 0x193d400e4 0x182284198 0x183138ed4 0x1004aecf8 0x1000c6d6c 0x1000c71dc 0x1000ed3f4 0x1000ecca8 0x183178180 0x18223c12c 0x18223bddc 0x182239828 0x1821652d4 0x18ba7b6fc 0x186d62f40 0x1000c5480 0x1943eaa08)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

**Comment by eunikolsky** [comment]
Fixed by @Andolamin as part of this commit: https://github.com/ConnectSDK/Connect-SDK-Cordova-API-Sampler/commit/65c6c3de8f098e529a5f47c687aa2d445d2e10c8#diff-0517663c4edf1f3aa9c1d36165349077L17  We'll close this bug once that branch is merged into `dev`.
